docs/03-tutorials/00-serverless/svls-11-set-asynchronous-connection-of-functions.md

```diff
@@ -0,0 +1,110 @@
+  
+  - `package.json` with the Function's dependencies
+
+4. In the `config.yaml` file configure your API Rule to allow for the incoming outside traffic (using HTTP). Enter your cluster domain in the host key:
+
+  ```yaml
+  apiRules:
+    - name: incomming-http-trigger
+      service:
+        host: incomming.inclusterevent.acme.shoot.canary.k8s-hana.ondemand.com
+      rules:
+        - methods:
+            - GET
+          accessStrategies:
+            - handler: allow
+  ```
+
+5. Provide your Function logic in the `handler.js` file. In the example you can find a simple code sanitization:
+
+   ```js
+   const { v4: uuidv4 } = require('uuid');
+   module.exports - {
+       main: function (event, context) {
+           let sanitsedData = sanitise(event.data)
+           var eventOut=event.buildResponseCloudEvent(uuidv4(), "sap.kyma.custom.acme.payload.sanitised.v1", sanitisedData);
+           eventOut.source="kyma"
+           eventOut.specversion="1.0"
+           event.publishCloudEvent(eventOut);
+           console.log(`Payload pushed to sap.kyma.custom.acme.payload.sanitised.v1`,eventOut)
+           return eventOut;
+       }
+   }
+   let sanitise = (data)=>{
+       console.log(`sanitising data...`)
+       console.log(data)
+       return data
+   }
+   ```
+
+6. Apply your emitter Function:
+
+  ```bash
+  kyma apply function
+  ```
+Your emitter Function sends the event, altered in a required way, to the receiver Function.
+
+7. To send the event you need to build the Cloud Event. To learn more please visit the [Function's Specification](https://kyma-project.io/docs/kyma/latest/05-technical-reference/svls-08-function-specification#event-object-sdk) page.
+In addition, your `eventOut.source` key needs to point at `“kyma”` to use Kyma In-cluster Eventing.
+
+8. Go to your `receiver` folder and run the `init` Kyma CLI command to initiate your function:
+   ```bash
+   kyma init function
+   ```
+9. The `init` command creates the same files as in the emitter folder.
+10. In the `config.yaml` file configure your Subscriptions to allow for receipt and storage of the event data:
+    ```yaml
+    name: event-receiver
+    namespace: default
+    runtime: nodejs14
+    source:
+       sourceType: inline
+    subscriptions:
+       - name: event-receiver
+         protocol: ""
+         filer:
+            filters:
+               - eventSource:
+                   property: source
+                   type: exact
+                   value: ""
+                eventType:
+                   property: type
+                   type: exact
+                   value: sap.kyma.custom.acme.payload.sanitised.v1
+    ```
+11. The `value` key in your `handler.js` file must consist of seven elements separated by dots, for example `sap.kyma.custom.acme.payload.sanitised.v1`. First three elements must remain unchanged (`sap.kyma.custom`). The last element specifies the version of the event.
```

Comment:
the event publisher is in ppower to pick event type. Event receiver must subscribe to whatever emiter chooses as event type.
So its better to explain it as part of the emmiter code ( above)